docs/api-reference/gltf/glb-loader.md

```diff
@@ -0,0 +1,30 @@
+# GLBLoader
+
+The `GLBLoader` parses a GLB binary "envelope".
+
+Note: applications that want to parse GLB-formatted glTF files use the `GLTFLoader` instead. The `GLBLoader` is intended to be used to load custom data that combines JSON and binary resources.
+
+| Loader                | Characteristic  |
+| --------------------- | --------------- |
+| File Extensions       | `.glb`          |
+| File Types            | Binary          |
+| Input Format          | [GLB](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification) |
+| Output Format         | [Scenegraph](/docs/specifications/category-scenegraph) |
```

Comment:
Saying that `GLBLoader` outputs a scenegraph is not consistent with the design, the `GLBLoader/GLBWriter` returns a very minimal object (a json structure and an array buffer) which does not need to have any relation to a scenegraph.

The idea was that GLB classes could be used for any hybrid JSON/binary format.

Might be worth just adding back the data format section here and just say "see below" in the output format row.